explorer/src/Pos/Explorer/TestUtil.hs

```diff
@@ -89,24 +71,56 @@ generateValidExplorerMockableMode blocksNumber slotsPerEpoch = do
 
     slotStart     <- liftIO $ generate $ arbitrary
 
-    blockHHs      <- liftIO $ generate $ arbitrary
-    blundsFHHs    <- liftIO $ generate $ withDefConfigurations arbitrary
-
     slotLeaders   <- produceSlotLeaders blocksNumber
     secretKeys    <- produceSecretKeys blocksNumber
 
     blocks <- withDefConfigurations $
         produceBlocksByBlockNumberAndSlots blocksNumber slotsPerEpoch slotLeaders secretKeys
 
-    let tipBlock = Prelude.last blocks
+    let tipBlock  = Prelude.last blocks
+    let pagedHHs  = withDefConfigurations $ createMapPageHHs blocks
+    let hHsBlunds = withDefConfigurations $ createMapHHsBlund blocks
 
     pure $ ExplorerMockableMode
         { emmGetTipBlock          = pure tipBlock
-        , emmGetPageBlocks        = \_ -> pure $ Just blockHHs
-        , emmGetBlundFromHH       = \_ -> pure $ Just blundsFHHs
-        , emmGetSlotStart         = \_ -> pure $ Just slotStart
-        , emmGetLeadersFromEpoch  = \_ -> pure $ Just slotLeaders
+        , emmGetPageBlocks        = \page -> pure $ pagedHHs ^. at page
+        , emmGetBlundFromHH       = \hh   -> pure $ hHsBlunds ^. at hh
+        , emmGetSlotStart         = \_    -> pure $ Just slotStart
+        , emmGetLeadersFromEpoch  = \_    -> pure $ Just slotLeaders
         }
+  where
+    createMapPageHHs :: (HasConfiguration) => [Block] -> Map Page [HeaderHash]
+    createMapPageHHs blocks =
+        fromListWith (++) [ (k, [v]) | (k, v) <- createPagedHeaderHashesPair blocks]
```

Comment:
I guess `k` is `key` and `v` is `value`. What do you think about renaming these as follow?
* `k` -> `page` 
* `v` -> `hash` or `headerHash`